docs/static/monitoring/collectors.asciidoc

```diff
@@ -3,11 +3,11 @@
 [[logstash-monitoring-collectors]]
 ===== Collectors
 
-Collectors, as their name implies, collect things. In {monitoring} for Logstash, 
+Collectors, as their name implies, collect things. In monitoring for Logstash, 
 collectors are just <<pipeline,Inputs>> in the same way that ordinary Logstash 
 configurations provide inputs.
 
-Like {monitoring} for {es}, each collector can create zero or more monitoring 
+Like monitoring for {es}, each collector can create zero or more monitoring 
```

Comment:
Any reason why the {monitoring} was switched to just monitoring?

Reply:
Yes. `{monitoring}` renders as "X-Pack monitoring," and we are trying to remove that terminology (as we can) across the Stack docs.  Headings currently carry the X-Pack badge that links to the licensing pages, so the distinction is still there. 

<img width="756" alt="Screen Shot 2020-04-09 at 11 06 00 AM" src="https://user-images.githubusercontent.com/35154725/78910325-c50b9900-7a52-11ea-91c0-30b097916324.png">